### DISCUSSION TITLE
Getting gmail to work for sending email

### DISCUSSION THREAD

**Discussion by abubin** [auanasgheps/snapraid-aio-script#90] [General]
Hi,

First of all, great script. Clear instructions and well structured coding.

For my question, do you have any guide for getting the email sending to work for those using gmail? I am not familiar with python. Not sure how to setup this. Thanks.

**Comment by auanasgheps** [comment]
Hi, my script relies on a 3rd party daemon to send emails. By default, uses on `mailx`, but can use any email service.

Me and many people use OpenMediaVault as base OS, which has its own GUI to configure the mail dameon (which is `mailx`). There are many guides on the internet to configure this service!

**Comment by tim-shortons** [comment]
@auanasgheps 

After updating packages my mailx instance was borked, nothing to do with the script, and I had to switch to mutt.  Mutt will not work with your script as some of the command line args being passed are not compatible.

Here is the change I needed to make to get mutt working with your script at line 990.  Note the UTF-8 charset is set in the muttrc config file and why I didn't pass it in the command.
`$MAIL_BIN -e "set content_type=text/html" -s "$SUBJECT" "$EMAIL_ADDRESS" `

**Reply by auanasgheps** [reply]
Hey, you could try the new notification agent called "apprise", from the latest beta. It allows [to send emails ](https://github.com/caronc/apprise?tab=readme-ov-file#email-notifications) using its own engine.
I know it's a beta but I will finish it off soon. Please let me know if you manage to make it working!
